doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -8,28 +8,274 @@ Get started fast for common scenarios, using Neo4j Streams plugin or Kafka Conne
+Neo4j Spark Connector allows Spark to read from and write to Neo4j databases.
 
-=== Spark Settings
+=== Getting Started
 
-(Setting detail here)
+Reading all the nodes of type `Person` from your local Neo4j instance is as simple as this:
+
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("labels", "Person")
+        .load()
+```
+
+=== Neo4j Options
+
+Spark method `option` is used to configure the Neo4j Connector.
+
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("authentication.type", "basic")
+        .option("authentication.basic.username", "myuser")
+        .option("authentication.basic.password", "neo4jpassword")
+        .option("labels", "Person")
+        .load()
+```
 
 .Most Common Needed Configuration Settings
 |===
-|Setting Name |Description |Default Value
+|Setting Name |Description |Default Value |Required 
+
+4+|*Driver Options*
+
+|`url`
+|The url of the Neo4j instance to connect to
+|_(none)_
+|Yes
+
+|`authentication.type`
+|The authentication method to be used: `none`, `basic`, `kerberos`, `custom`. More info https://neo4j.com/docs/driver-manual/4.1/client-applications/#driver-authentication[here]
+|`basic`
+|No
+
+|`authentication.basic.username`
+|Username to use for basic authentication type
+|``
+|No
+
+|`authentication.basic.password`
+|Username to use for basic authentication type
+|``
+|No
+
+|`authentication.kerberos.ticket`
+|Kerberos Auth Ticket
+|``
+|No
+
+|`authentication.custom.principal`
+|This used to identify who this token represents
+|``
+|No
+
+|`authentication.custom.credentials`
+|This are the credentials authenticating the principal
+|``
+|No
+
+|`authentication.custom.realm`
+|This is the "realm:"", specifying the authentication provider
+|``
+|No
+
+|`authentication.custom.schema`
+|This it the authentication scheme, specifying what kind of authentication that should be used
+|``
+|No
+
+|`encryption.enabled`
+|Specify if encryption should be enabled
+|`false`
+|No
+
+|`encryption.trust.strategy`
+|Set cerificate trust strategy
+|`TRUST_SYSTEM_CA_SIGNED_CERTIFICATES`
+|No
+
+|`encryption.ca.certificate.path`
+|Set certificate path for `TRUST_CUSTOM_CA_SIGNED_CERTIFICATES` trust strategy
+|``
+|No
+
+|`connection.max.lifetime.msecs`
+|Connection lifetime in milliseconds
+|(Neo4j Driver default)
+|No
+
+|`connection.liveness.timeout.msec`
+|Liveness check timeout
+|(Neo4j Driver default)
+|No
+
+|`connection.acquisition.timeout.msecs`
+|Connection acquisition timeout in milliseconds
+|(Neo4j Driver default)
+|No
+
+|`connection.timeout.msecs`
+|Connection timeout in milliseconds
+|(Neo4j Driver default)
+|No
+
+
+4+|*Session Options*
+
+|`database`
+|Database name to connect to
+|``
+|No
+
+|`access.mode`
+|Access mode
+|`write`
+|No
+
+4+|*Data Read Options*
+
+|`query`
+|Specify a query to read the data
+|``
+|Yes^*^
+
+|`labels`
```

Comment:
I did this: 

```
  .option("labels", ":City") \
```

Maybe bit a bit more tolerant because this is a very natural thing for a Neo4j user to do, although technically we're passing a blank label, I know.

```
org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 12.0 failed 1 times, most recent failure: Lost task 0.0 in stage 12.0 (TID 12, localhost, executor driver): java.lang.IllegalArgumentException: A primary label is required.
```

Reply:
done